integration-test/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesTestComponents.scala

```diff
@@ -56,6 +57,8 @@ private[spark] class KubernetesTestComponents(defaultClient: DefaultKubernetesCl
     new SparkAppConf()
       .set("spark.master", s"k8s://${kubernetesClient.getMasterUrl}")
       .set("spark.kubernetes.namespace", namespace)
+      // TODO: apache/spark#19995 is changing docker.image to container.image in these properties.
```

Comment:
FYI: the PR has been merged.

Reply:
I see. I'll have to update my distro, make this change and test it again.

Reply:
This is done by commit 989a371.